device/api/umd/device/cluster.h

```diff
@@ -823,8 +823,8 @@ class Cluster : public tt_device {
     virtual std::uint32_t get_host_channel_size(std::uint32_t device_id, std::uint32_t channel);
     virtual std::uint32_t get_numa_node_for_pcie_device(std::uint32_t device_id);
     virtual tt_version get_ethernet_fw_version() const;
-    // TODO: This should be accessible through public API, probably to be moved to tt_device.
-    PCIDevice* get_pci_device(int device_id) const;
+
+    TTDevice* get_tt_device(int device_id) const;
```

Comment:
TTDevice abstraction only exists for PCIe-attached chips, correct?  Should this be clear from the interface?

Reply:
Yes, it will be. Look at https://github.com/tenstorrent/tt-umd/pull/351
The LocalChip (and not remotechip) will eventually hold this pointer, so it will be clear.